### DISCUSSION TITLE
UQL Parser: How to address entire result row?

### DISCUSSION THREAD

**Discussion by CoreMoni** [grafana/grafana-infinity-datasource#745] [Q&A] [answered]
Hi,

my datasource returns a CSV file with semicolon-separated values, not comma-separated.
With the UQL parser I get the entire row as one value. I found that there is a split function that I could use to separate the values, but what do I need to enter for "?" to address the entire result row?

`parse-csv
| project "tabrow"=split("?",';')
| extend "col0"="tabrow[0]", "col1"="tabrow[1]"`

Thanks

**Comment by yesoreyeram** [comment, marked as answer]
I think you needed https://github.com/grafana/grafana-infinity-datasource/discussions/393#discussioncomment-3600006

**Reply by CoreMoni** [reply]
This works perfectly, thank you!
